src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/adjudications/CreateAdjudicationRequest.kt

```diff
@@ -33,22 +33,25 @@ data class ChargeToCreate(
 
 @JsonInclude(JsonInclude.Include.NON_NULL)
 data class IncidentToCreate(
-  @Schema(description = "Reporting staff member username", required = true, example = "JANE.BROOKES")
+  @Schema(description = "Reporting staff member username", example = "JANE.BROOKES")
   val reportingStaffUsername: String,
 
-  @Schema(description = "Date of the associated incident", required = true)
+  @Schema(description = "Date of the associated incident")
   val incidentDate: LocalDate,
 
-  @Schema(description = "Date and time of the associated incident", required = true, example = "12:00:00")
+  @Schema(description = "Date and time of the associated incident", example = "12:00:00")
   val incidentTime: LocalTime,
 
-  @Schema(description = "Date when the associated incident was reported", required = true)
```

Comment:
required is deprecated and of course it can work it out from nullability in Kotlin